docs/team/weikiat.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Wei Kiat's Project Portfolio Page
+---
+// Still in progress //
+### Project: AddressBook Level 3
+
+AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **Code contributed**: [RepoSense link]()
```

Comment:
Maybe you can consider adding the RepoSense link from the code dashboard in the module website?